### PR TITLE
temporary schedules: rename UI bits to match new naming

### PR DESCRIPTION
**Pull request by mastercactapus**
- [x] Identified the issue which this PR solves.
- [x] Read the [**CONTRIBUTING**](https://github.com/target/goalert/blob/master/CONTRIBUTING.md) document.
- [x] Code builds clean without any errors or warnings.
- [x] Added appropriate tests for any new functionality.
- [x] All new and existing tests passed.
- [x] Added comments in the code, where necessary.
- [x] Ran `make check` to catch common errors. Fixed any that came up.

**Description:**
This PR updates UI variables, comments, and component names to be consistent with the TemporarySchedules naming.

**Which issue(s) this PR fixes:**
Part of #862

### CODE REVIEW

**Comment by m17ch** [comment]
image


Getting these errors in console when functionally validating (viewing any schedule):
```
ScheduleCalendar.js:242 Uncaught TypeError: Cannot read property 'map' of undefined
    at ScheduleCalendar._this.getCalEvents (ScheduleCalendar.js:242)
    at ScheduleCalendar.render (ScheduleCalendar.js:197)
    at finishClassComponent (react-dom.development.js:17185)
    at updateClassComponent (react-dom.development.js:17135)
    at beginWork (react-dom.development.js:18654)
    at HTMLUnknownElement.callCallback (react-dom.development.js:189)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:238)
    at invokeGuardedCallback (react-dom.development.js:293)
    at beginWork$1 (react-dom.development.js:23235)
    at performUnitOfWork (react-dom.development.js:22186)
```

```
The above error occurred in the <ScheduleCalendar> component:
    in ScheduleCalendar (created by ConnectFunction)
    in ConnectFunction (created by WithStyles(Connect(ScheduleCalendar)))
    in WithStyles(Connect(ScheduleCalendar)) (created by Query)
    in Query (created by Query)
    in Query (created by ScheduleCalendarQuery)
    in ScheduleCalendarQuery (created by ConnectFunction)
    in ConnectFunction (created by WithWidth)
    in WithWidth (created by ScheduleDetails)
    in div (created by ForwardRef(Grid))
    in ForwardRef(Grid) (created by WithStyles(ForwardRef(Grid)))
    in WithStyles(ForwardRef(Grid)) (created by DetailsPage)
    in div (created by ForwardRef(Grid))
    in ForwardRef(Grid) (created by WithStyles(ForwardRef(Grid)))
    in WithStyles(ForwardRef(Grid)) (created by DetailsPage)
    in DetailsPage (created by ScheduleDetails)
    in ScheduleDetails (created by Context.Consumer)
    in Route (created by ScheduleRouter)
    in Switch (created by ScheduleRouter)
    in ScheduleRouter (created by Context.Consumer)
    in Route (created by App)
    in Switch (created by App)
    in div (created by ForwardRef(Grid))
    in ForwardRef(Grid) (created by WithStyles(ForwardRef(Grid)))
    in WithStyles(ForwardRef(Grid)) (created by App)
    in div (created by ForwardRef(Grid))
    in ForwardRef(Grid) (created by WithStyles(ForwardRef(Grid)))
    in WithStyles(ForwardRef(Grid)) (created by App)
    in ErrorBoundary (created by App)
    in main (created by App)
    in SearchProvider (created by App)
    in PageActionProvider (created by App)
    in div (created by App)
    in App (created by ConnectFunction)
    in ConnectFunction (created by WithMobileDialog)
    in WithMobileDialog (created by WithWidth(WithMobileDialog))
    in WithWidth(WithMobileDialog) (created by WithStyles(WithWidth(WithMobileDialog)))
    in WithStyles(WithWidth(WithMobileDialog))
    in GracefulUnmounterProvider
    in ConfigProvider
    in MuiPickersUtilsProvider (created by PickersUtilsProvider)
    in PickersUtilsProvider
    in Router (created by ConnectedRouter)
    in ConnectedRouter (created by Context.Consumer)
    in ConnectedRouterWithContext (created by ConnectFunction)
    in ConnectFunction
    in Provider
    in ApolloProvider
    in ThemeProvider

React will try to recreate this component tree from scratch using the error boundary you provided, ErrorBoundary.
```

```
TypeError: Cannot read property 'map' of undefined
    at ScheduleCalendar._this.getCalEvents (ScheduleCalendar.js:242)
    at ScheduleCalendar.render (ScheduleCalendar.js:197)
    at finishClassComponent (react-dom.development.js:17185)
    at updateClassComponent (react-dom.development.js:17135)
    at beginWork (react-dom.development.js:18654)
    at HTMLUnknownElement.callCallback (react-dom.development.js:189)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:238)
    at invokeGuardedCallback (react-dom.development.js:293)
    at beginWork$1 (react-dom.development.js:23235)
    at performUnitOfWork (react-dom.development.js:22186) {componentStack: "↵    in ScheduleCalendar (created by ConnectFuncti…ovider↵    in ApolloProvider↵    in ThemeProvider"} "
    at ErrorBoundary (http://localhost:3030/static/app.js:12840:81)
    at main
    at SearchProvider (http://localhost:3030/static/app.js:31618:81)
    at PageActionProvider (http://localhost:3030/static/app.js:34186:81)
    at div
    at App (http://localhost:3030/static/app.js:12701:81)
    at ConnectFunction (webpack://%5Bname%5D/./node_modules/react-redux/es/components/connectAdvanced.js?:231:75)
    at WithMobileDialog (webpack://%5Bname%5D/./node_modules/@material-ui/core/esm/withMobileDialog/withMobileDialog.js?:26:102)
    at WithWidth(WithMobileDialog) (webpack://%5Bname%5D/./node_modules/@material-ui/core/esm/withWidth/withWidth.js?:59:90)
    at WithStyles (webpack://%5Bname%5D/./node_modules/@material-ui/styles/esm/withStyles/withStyles.js?:61:31)
    at GracefulUnmounterProvider (http://localhost:3030/static/app.js:36037:81)
    at ConfigProvider (http://localhost:3030/static/app.js:34811:23)
    at MuiPickersUtilsProvider (webpack://%5Bname%5D/./node_modules/@material-ui/pickers/esm/useUtils-cfb96ac9.js?:14:20)
    at PickersUtilsProvider (http://localhost:3030/static/app.js:14571:12)
    at Router (webpack://%5Bname%5D/./node_modules/react-router/esm/react-router.js?:93:30)
    at ConnectedRouter (webpack://%5Bname%5D/./node_modules/connected-react-router/esm/ConnectedRouter.js?:56:7)
    at ConnectedRouterWithContext (webpack://%5Bname%5D/./node_modules/connected-react-router/esm/ConnectedRouter.js?:154:25)
    at ConnectFunction (webpack://%5Bname%5D/./node_modules/react-redux/es/components/connectAdvanced.js?:231:75)
    at Provider (webpack://%5Bname%5D/./node_modules/react-redux/es/components/Provider.js?:14:20)
    at ApolloProvider (webpack://%5Bname%5D/./node_modules/@apollo/react-common/lib/react-common.esm.js?:27:21)
    at ThemeProvider (webpack://%5Bname%5D/./node_modules/@material-ui/styles/esm/ThemeProvider/ThemeProvider.js?:42:24)"
```

**Comment by mastercactapus** [comment]
@m17ch apologies -- fixed now, tested create, edit, & delete

**Review by m17ch — DISMISSED** [review]
Thanks, @mastercactapus -- LGTM!

File: web/src/app/schedules/ScheduleCalendar.js [hunk 1]
```diff
@@ -241,27 +238,27 @@ export default class ScheduleCalendar extends React.PureComponent {
     )
   }
 
-  getCalEvents = (shifts, _fixedShifts) => {
-    const fixedSchedules = _fixedShifts.map((sched) => ({
+  getCalEvents = (shifts, _tempScheds) => {
+    const TempSchedules = _tempScheds.map((sched) => ({
```
Comment (Forfold) [inline web/src/app/schedules/ScheduleCalendar.js:242]: nit:
```suggestion
    const tempSchedules = _tempScheds.map((sched) => ({
```
and references

**Review by m17ch — APPROVED** [review]

**Pull request merged**